roles/dg-config/tasks/main.yml

```diff
@@ -0,0 +1,198 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: DG | SQL to add standby logs
+  set_fact:
+    sql_for_dg:  |
+      alter system set dg_broker_start = true;
+      alter system set standby_file_management = auto;
+      begin
+      for c in (
+        select bytes, thr, (max(cnt_log) - nvl(max(cnt_sby), 0)) diff from (
+        select bytes, thread# thr, count(*) cnt_log, null cnt_sby from v\$log where thread# = 1 group by bytes, thread#
+        union all
+        select bytes, thread# thr, null cnt_log, count(*) cnt_sby from v\$standby_log where thread# = 1 group by bytes, thread#
+        )
+        group by bytes, thr
+      ) loop
+        for n in 1..c.diff + 1 loop
+          execute immediate 'alter database add standby logfile thread '||c.thr||' size '||c.bytes;
+        end loop;
+      end loop;
+      end;
+      /
+  tags: dg-create
+
+- name: DG | Change primary database for DG setup
+  # start DG broker
```

Comment:
I think this step only enables force logging

Reply:
commented on it below